pstats_print2list/pstats_print2list.py

```diff
@@ -5,7 +5,16 @@
 import os
 import pstats
 import re
-import StringIO
+try:
+    import StringIO
+except ImportError:
+    from io import StringIO
+
+
+try:
+    basestring
+except NameError:
+    basestring = str
```

Comment:
Use typestring like as your use it from pylint-odoo

Reply:
Done